packages/components/src/base-control/README.md

```diff
@@ -63,6 +63,14 @@ The content to be displayed within the BaseControl.
 -   Type: `Element`
 -   Required: Yes
 
+### __nextHasNoMarginBottom
+
+-   Type: `Boolean`
+-   Required: No
+-   Default: `false`
+
+Start opting into the new margin-free styles that will become the default in a future version.
```

Comment:
Nit: usually the description comes before the prop "type" definitions

```suggestion
Start opting into the new margin-free styles that will become the default in a future version.

-   Type: `Boolean`
-   Required: No
-   Default: `false`
```

Reply:
Good catch, thanks.